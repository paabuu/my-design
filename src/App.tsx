import * as React from 'react';
import Button from '../components/button';
import Alert from '../components/alert';
import Affix from '../components/affix';
import Checkbox from '../components/checkbox';
import Radio from '../components/radio';

import '../components/style/index.less';
import './App.less';

const { useState } = React;
const RadioGroup = Radio.Group;

const App = () => {
    const [checked, setChecked] = useState(false);
    const [value, setValue] = useState('r2');
    
    return (
        <div>
            {/* <Affix>
                <Button type="primary">Primary</Button>
            </Affix> */}
            {/* <Button>чбошод</Button>
            <Button type="dashed">Dashed</Button>
            <Button type="danger" onClick={e => alert('danger!')}>Danger</Button>
            <Alert message="hahah" description="xixixixixix "></Alert>
            <Alert type="success" message="hahah" description="xixixixixix "></Alert>
            <Alert type="warning" message="hahah" description="xixixixixix "></Alert>
            <Alert type="error" message="hahah"></Alert> */}
            {/* <Checkbox 
                checked={true}
            >Checkbox</Checkbox> */}
            <RadioGroup 
                value={value} 
                onChange={e => setValue(e.target.value)} 
                size="small"
            >
                <Radio value="r1" style={{marginRight: 10}} >Radio1</Radio>
                <Radio value="r2">Radio2</Radio>
            </RadioGroup>
        </div>
    );
};

export default App;