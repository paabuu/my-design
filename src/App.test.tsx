import * as React from 'react';
import { render } from 'enzyme';
import App from './App';

it('app mounted', () => {
    const app = render(<App />);
    expect(app).toMatchSnapshot();
});