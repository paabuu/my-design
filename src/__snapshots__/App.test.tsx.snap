// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app mounted 1`] = `
<div>
  <div
    class="md-radio-group"
  >
    <label
      class="md-radio-wrapper"
      style="margin-right:10px"
    >
      <span
        class="md-radio md-radio-small"
      >
        <input
          type="radio"
          value="r1"
        />
        <span
          class="md-radio-inner"
        />
      </span>
      <span
        style="cursor:pointer"
      >
        Radio1
      </span>
    </label>
    <label
      class="md-radio-wrapper"
    >
      <span
        class="md-radio md-radio-small md-radio-checked"
      >
        <input
          checked=""
          type="radio"
          value="r2"
        />
        <span
          class="md-radio-inner"
        />
      </span>
      <span
        style="cursor:pointer"
      >
        Radio2
      </span>
    </label>
  </div>
</div>
`;
